schema.sql

```diff
@@ -1,5 +1,9 @@
-/* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE
+    animals (
+        id SERIAL PRIMARY KEY,
```

Comment:
[OPTIONAL]
- [ ] I would advice that you make the *id column not null and autoincremented*, see the below snippet as a guide
`id int NOT NULL IDENTITY(1,1)`

Reply:
Hello, Thank you for your review. I tried that one at first actually but it didn't work out. I researched the solution for it and it is because the `IDENTITY` data type is not supported in PostgreSQL. They said the query is specific to Microsoft SQL Server.